packages/rpc-graphql/src/cache.ts

```diff
@@ -0,0 +1,63 @@
+export interface GraphQLCache {
+    flush(): void;
+    get(key: string | bigint, variables: unknown): unknown | null;
+    insert(key: string | bigint, variables: unknown, value: unknown): void;
+}
+
+// Basic in-memory cache for Node.js
+const inMemoryCache: { [key: string]: string } = {};
+
+const stringifyValue = (value: unknown) =>
+    JSON.stringify(value, (_, value) => {
+        if (typeof value === 'bigint') {
+            return value.toString() + 'n';
+        }
+        return value;
```

Comment:
Nitpick: shadowing `value` here (if I'm understanding it correctly) is quite confusing! Same on `parseValue`

Reply:
Yep totally fair

Reply:
Although technically that _is_ what `JSON.stringify(..)` does, too! I can change it though.